report_pre_thesis/src/sections/experiments.tex

```diff
@@ -112,30 +66,74 @@ \subsection{Replication of the MOC Pipeline}\label{sec:replica_moc}
 	\label{fig:rmse_histograms}
 \end{figure*}
 
-To evaluate the statistical significance of the differences between the original and our replicated models (PLS1-SM, ICA, and MOC), we applied three statistical tests: Student's t-test, Welch's t-test, and the Wilcoxon signed-rank test.
-Our null hypothesis (H$_0$) posits that there is no significant difference in the performance of the original and replicated models.
-The alternative hypothesis (H$_1$) contends that a significant difference does exist.
-We set our significance level $\alpha$ at 0.05, meaning we would require a p-value less than 0.05 to reject the null hypothesis in favor of the alternative.
-The p-values obtained from our tests are presented in Table~\ref{tab:results_ttests}. With all p-values exceeding the $\alpha$ level of 0.05, there is insufficient statistical evidence to reject the null hypothesis.
-Therefore, we retain the null hypothesis, suggesting that the differences between the original and replicated models are not statistically significant.
-The p-values across all tests and models imply that the performances of the original and replicated models are statistically indistinguishable from each other.
-This congruence suggests that our replicated models are comparable to the original ones in terms of their statistical characteristics.
+\citet{andersonImprovedAccuracyQuantitative2017} used the Student's t-test to show their new model outperformed the old one. In our study, we apply the same test but with a different aim: to verify if our replicated models (PLS1-SM, ICA, and MOC) are statistically similar to the original MOC  models, presented in \citet{cleggRecalibrationMarsScience2017}. This approach allows us to assess whether our models demonstrate equivalence rather than improvement.
```

Comment:
```suggestion
\citet{andersonImprovedAccuracyQuantitative2017} used the Student's t-test to show their new model outperformed the old one. In our study, we apply the same test but with a different aim: to verify if there is no statistical significant difference between our replicated models (PLS1-SM, ICA, and MOC) and the original models presented in \citet{cleggRecalibrationMarsScience2017}. This approach allows us to assess whether our models demonstrate equivalence rather than improvement.
```